pkg/simulation/target_test.go

```diff
@@ -23,7 +23,54 @@ func NewSim(t *testing.T, seed int64) (*simulation.Simulation, *mock.MockAttribu
 	return sim, attr
 }
 
-func TestRetargetSimple(t *testing.T) {
+// test var : Targets list
+func TestRetargetEmptyTargets(t *testing.T) {
+	sim, attr := NewSim(t, 1)
+
+	// for all targets, return full health in this test
+	attr.EXPECT().HPRatio(gomock.Any()).Return(1.0).AnyTimes()
+
+	result := sim.Retarget(info.Retarget{
+		Targets: []key.TargetID{},
+		Filter:  func(target key.TargetID) bool { return target == 2 },
+		Max:     3,
+	})
+
+	assert.Equal(t, []key.TargetID{}, result)
```

Comment:
Make sure you are using the right/best assert for your test. In this case, `assert.Empty(t, result)` makes more sense here